README.md

```diff
@@ -138,11 +139,55 @@ To specify additional optional tool-specific parameters, see what flags are supp
 ```
 nextflow run main.nf --help 
 ```
+A more advanced nextflow command with optional parameters, where you can define the name of the output folder and provide an integer value for the number of genomic-interval files to be generated looks like this:
+```
+nextflow run main.nf --input samples.csv \
+                     --ref /path/to/ref.fasta --dict /path/to/ref.dict \
+                     --ponvcf /path/to/pon \
+                     --common_biallelic_variants /path/to/common_biallelic_variants \
+                     --outDir name_output_folder \
+                     --number_of_intervals INT
+```
+
+**Mandatory parameters**  
+* `--input`		                      Full path and name of sample input file (csv format)  
+* `--dict`                          Full path and name of reference genome dictionary file (dict format)  (**Step:** create genomic-intervals)  
+* `--ref`			                      Full path and name of reference genome (fasta format)                 (**Step:** Mutect2)  
+* `--ponvcf`                        Full path and name of the Panel of Normals file (vcf format)          (**Step:** Mutect2)  
+* `--common_biallelic_variants`     Full path and name of the common biallelic variant resources file (vcf format)(**Step:** GetPileupSummaries)  
+
+**Optional parameters**  
+* `--outDir`                        Name of the results directory                                         (**default:** `results`)  
+* `--number_of_intervals`           Define a specific number genomic-intervals for parallelisation        (**default:** `number of intervals are calculated using genome size`)  
+
 
 If for any reason your workflow fails, you are able to resume the workflow from the last successful process with `-resume`. 
 
+This pipeline has been optimised for NCI Gadi HPC, instructions for executing on Gadi are provided in [Infrastructure usage and recommendations](#infrastructure-usage-and-recommendations)
+
+### 5. Results 
+Once the pipeline is complete, you will find all outputs for each sample in the `results` directory. 
```

Comment:
Going to create a separate issue for this so it can be addressed in a new pull request, we need to explain what the outputs are and how they are organised.